upa/contracts/UpaVerifier.sol

```diff
@@ -583,7 +605,7 @@ contract UpaVerifier is
 
         bytes32 submissionId = UpaLib.computeSubmissionId(proofId);
 
-        uint64 submissionIdx = getSubmissionIdx(submissionId);
+        uint64 submissionIdx = getSubmissionIdx(submissionId, 0);
```

Comment:
I think this `isProofVerified` needs to be updated to also check the later dupSubmissionIdxs.

Can it just call `isSubmissionVerified(submissionId)`? I don't remember if there was a reason I didn't implement that way.

Reply:
Thanks.  Good catch.  Yes, I can't think of a reason not to, now.  All tests pass so I'll do it that way.